packages/serverpod/lib/src/server/websocket_request_handlers/helpers/method_stream_manager.dart

```diff
@@ -336,9 +340,12 @@ class MethodStreamManager {
       /// or a request from the client.
       if (isCancelled) return;
       isCancelled = true;
+      session.serverpod.logVerbose(
+          'Cancelling method output stream for ${session.endpoint}.${session.method}, id $methodStreamId');
       await revokedAuthenticationHandler?.destroy(session);
       await _closeOutboundStream(methodStreamCallContext, methodStreamId);
-      await session.close();
+      await session
+          .close(); // Will this also close the input streams? If so is this the desired behavior?
```

Comment:
Suggestion: Remove comment.

Answer to the comment: These are closed in the _closeOutboundStream method and yes, this is the desired behavior. The lifecycle of a stream is determined by the output stream. If it is closed, all input streams should also be closed.

Reply:
Thx!